### PR TITLE
chore(demo): new Jest setup documentation

### PR DESCRIPTION
**Pull request by Yohandah**
## PR Checklist

Addresses https://github.com/Tinkoff/taiga-ui/issues/2337
I am not sure about the `keywords` part, I saw there was Russian there, I deleted it, I added what I thought could help people Googling the issue to find this page.

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

There's no documentation for using Jest with Taiga UI

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Comment by splincode** [comment]
image

**Comment by Yohandah** [comment]
Sorry I'm not sure why the Cypress test fails, there's no `tui-doc-example`  yes

**Comment by nsbarsukov** [comment]
I think we can exclude all `Testing` section from `demo`-tests.

@Yohandah could you add `Testing` to this array, please ?
https://github.com/Tinkoff/taiga-ui/blob/8f0c6db3611b27ecdbdc711d87c0b8fe1a61198d/projects/demo-integrations/cypress/support/demo-paths.ts#L7

File: projects/demo/src/modules/app/pages.ts [hunk 1]
```diff
@@ -42,6 +42,12 @@ export const pages: TuiDocPages = [
         section: $localize`Documentation`,
         title: $localize`Testing`,
         subPages: [
+            {
+                section: $localize`Jest`,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/pages.ts:46]: I have found error (why e2e fails). It here :)
```suggestion
                section: $localize`Documentation`,
```


My mistake here, sorry. It is useless. Could you revert this line
https://github.com/Tinkoff/taiga-ui/pull/2432#issuecomment-1223682621

Comment (nsbarsukov) [reply projects/demo/src/modules/app/pages.ts:46]: Or we should rename all sections inside this array to `Testing` (and dont revert this comment https://github.com/Tinkoff/taiga-ui/pull/2432#issuecomment-1223682621) 🤔

Comment (nsbarsukov) [reply projects/demo/src/modules/app/pages.ts:46]: I think **semantically** the second approach (rename all sections inside this array to `Testing`) is better.

@splincode are you agree ?

**Review by nsbarsukov — APPROVED** [review]
Thanks for your PR 🚀

**Pull request merged**